script.js

```diff
@@ -0,0 +1,108 @@
+let storedNotes = JSON.parse(localStorage.getItem("notes"));
+console.log(storedNotes);
+let notes = storedNotes ? storedNotes : [];
+let list = document.getElementById("list");
+console.log(notes);
+showNotes();
+
+document.getElementById("add-btn").addEventListener("click", function () {
```

Comment:
if the user tried to add input with just `spaces` on it, and no characters, it will add an empty list item. for example, try to press `space` more than one time on the input field and see what will happen.
It would be great if you handle this case!